.travis.yml

```diff
@@ -6,20 +6,24 @@ go:
 services:
   - rabbitmq
 
+sudo: required
```

Comment:
Alright, just wanted to make sure we were getting benefits and not just the slower start-times. :)